pkg/activator/net/revision_backends.go

```diff
@@ -301,6 +317,12 @@ func (rw *revisionWatcher) checkDests(curDests, prevDests dests) {
 		}
 	}
 
+	if rw.usePassthroughLb {
```

Comment:
👍️ I like that we'll get a clear indication here if the flag was set and doesnt work, rather than silently falling back and succeeding.